Makefile

```diff
@@ -1 +1,84 @@
-target: ;
\ No newline at end of file
+PROJECTNAME := $(shell basename "$(PWD)")
+
+# Go related variables.
+PROJECTROOT := $(shell pwd)
+GOBIN := $(PROJECTROOT)/bin
```

Comment:
Can you make sure this directory always exists?
Else `make build-cli' will fail.

Reply:
yeah missed this. there should be a `mkdir bin` in the beginning.